presto-mongodb/src/main/java/io/prestosql/plugin/mongodb/MongoSession.java

```diff
@@ -141,15 +144,18 @@ public void shutdown()
 
     public List<String> getAllSchemas()
     {
-        return ImmutableList.copyOf(client.listDatabaseNames());
+        return ImmutableList.copyOf(client.listDatabaseNames()).stream()
```

Comment:
`ImmutableList.copyOf` is redundant

Reply:
`client.listDatabaseNames()` returns `MongoIterable<String>` (≠`List<String>`).

Reply:
you have `ImmutableList.copyOf` and `collect(toImmutableList())`

Reply:
Replaced with `toList()`. Please let me know if my understanding is wrong.

Reply:
Was this change saved? I still see `collect(toImmutableList()) `

Reply:
@sajjoseph `collect(toImmutableList())` is the expected code.

Reply:
Thanks. I am referring to your comment earlier where you said that you replaced with toList(). I don't see that change.